lib/Fez/Util/Config.rakumod

```diff
@@ -2,16 +2,48 @@ unit module Fez::Util::Config;
 
 use Fez::Util::Json;
 
-state $CONFIG-PATH = (%*ENV<FEZ_CONFIG>//%?RESOURCES<config.json>).IO;
-state $CONFIG      = from-j($CONFIG-PATH.slurp);
+state $FALLBACK-CONFIG-PATH = (%*ENV<FEZ_FALLBACK_CONFIG>//%?RESOURCES<config.json>).IO;
+state %FALLBACK-CONFIG  = from-j($FALLBACK-CONFIG-PATH.slurp);
+state $USER-CONFIG-PATH = (
+  if $*DISTRO.is-win {
+    %*ENV<FEZ_CONFIG> //
+    %*ENV<APPDATA>.IO.add('fez').add('fez-config.json')
+  }
+  elsif run('xdg-user-dir', :out, :err).exitcode == 0 {
```

Comment:
I'm going to remove this block and merge.  If someone is messing with their $HOME variable then they likely know enough to figure out what this is doing and I don't like the idea of relying on `xdg-`.  The right fix is a C binding to `getpwuid` and moarvm doesn't have one.

Reply:
Did you notice my comment explaining these changes here? https://github.com/tony-o/raku-fez/pull/4#discussion_r549493435